src/blocks/ImageTiles/ImageTiles.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import Ublock from '@src/components/Ublock/Ublock'
+
+export interface ImageTilesProps {
+    uBlockFull: boolean
+    mainData: ImageTilesMainProps
+    subData: ImageTilesSubProps[]
```

Comment:
There's no subdata. 
All the date should come from the data props :)

Reply:
Done

Reply:
All the data should be an array and the first item should be the bigger image :)

Reply:
fixed, now the data are in an array